test/ping-req-tests.js

```diff
@@ -28,7 +28,12 @@ var getClusterSizes = require('./it-tests').getClusterSizes;
 
 test2('ping-req real-node with a disabled target', getClusterSizes(2), 20000,
     prepareCluster(function(t, tc, n) { return [
-        dsl.disableNode(t, tc, 1),
+        // By disabling all nodes, it's impossible for the real node to do an
+        // indirect health check itself. This denies the real node to make the
+        // target suspect. This is important because we check that the real
+        // node doesn't change the status of the target when sending a ping-req
+        // ping.
+        dsl.disableAllNodesPing(t, tc),
```

Comment:
Our goal in this test is to make a Fake node (`F1`) do a `ping-req` through the Real (`R`) node to Fake node (`F2`), and see the Real node correctly does the `ping-req`: returns `ok` if the `F2` is healthy, and `not ok` if `F2` is not healthy.

However, it can happen that Real node will send two more `ping-req` to 2 other Fake nodes (`F3` and `F4`), which will cause `R` to declare `F2` as suspect, which will interfere with the test. We don't want that.

To avoid `R` declaring `F2` a suspect, we can disable `ping` and `ping-req` in `F3` and `F4` (i.e., disable ping in all the Fake nodes).